src/main/java/pl/tomaszdziurko/jvm_bloggers/utils/Validators.java

```diff
@@ -0,0 +1,14 @@
+package pl.tomaszdziurko.jvm_bloggers.utils;
+
+import org.apache.commons.validator.routines.UrlValidator;
+
+public class Validators {
+
+    private static final UrlValidator URL_VALIDATOR =
+        new UrlValidator(new String[]{"http", "https"});
+
+    public static boolean isUrlValid(String url) {
+        return URL_VALIDATOR.isValid(url);
```

Comment:
\[Checkstyle\] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4\.